### PR TITLE
IGVF-860-rename-construct-library-origins

### PR DESCRIPTION
**Pull request by KAndreeva**
Renamed construct_library origins to selection_criteria.

### CODE REVIEW

**Review by gabdank — APPROVED** [review]

**Review by gabdank — APPROVED** [review]

File: src/igvfd/schemas/construct_library.json [hunk 1]
```diff
@@ -140,14 +140,14 @@
...
-            "title": "Origins",
+        "selection_criteria": {
+            "title": "Selection Criteria",
             "description": "The criteria used to select the sequence material cloned into the library.",
             "type": "array",
             "minItems": 1,
             "uniqueItems": true,
             "items": {
-                "title": "Origin",
+                "title": "Selection Criteria",
```
Comment (gabdank) [inline src/igvfd/schemas/construct_library.json:150]: criterion rather than criteria

File: src/igvfd/schemas/construct_library.json [hunk 2]
```diff
@@ -140,14 +140,14 @@
...
                 "description": "The criteria used to select or design the sequence material cloned into the library.",
```
Comment (gabdank) [inline src/igvfd/schemas/construct_library.json:151]: criterion

File: src/igvfd/schemas/construct_library.json (hunk 1 above)
Comment (gabdank) [reply src/igvfd/schemas/construct_library.json:150]: singular item, vs plural name for a list

**Review by gabdank — CHANGES_REQUESTED** [review]

File: src/igvfd/schemas/construct_library.json (hunk 1 above)
Comment (KAndreeva) [reply src/igvfd/schemas/construct_library.json]: Title updated.

File: src/igvfd/schemas/construct_library.json (hunk 2 above)
Comment (KAndreeva) [reply src/igvfd/schemas/construct_library.json]: Description updated.

**Review by gabdank — APPROVED** [review]

**Pull request merged**